module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -678,14 +707,15 @@ protected function getHoldingDetailsForItem($holding): array
      * Support method for getHolding() -- given a few key details, format an item
      * for inclusion in the return value.
      *
-     * @param string $bibId            Current bibliographic ID
-     * @param array  $holdingDetails   Holding details produced by
-     *                                 getHoldingDetailsForItem()
-     * @param object $item             FOLIO item record (decoded from JSON)
-     * @param int    $number           The current item number (position within
-     *                                 current holdings record)
-     * @param string $dueDateValue     The due date to display to the user
-     * @param array  $boundWithRecords Any bib records this holding is bound with
+     * @param string         $bibId            Current bibliographic ID
+     * @param array          $holdingDetails   Holding details produced by
+     *                                         getHoldingDetailsForItem()
+     * @param object         $item             FOLIO item record (decoded from JSON)
+     * @param int            $number           The current item number (position within
+     *                                         current holdings record)
+     * @param string         $dueDateValue     The due date to display to the user
+     * @param array          $boundWithRecords Any bib records this holding is bound with
+     * @param \stdClass|void $currentLoan      Any current loan on this item
```

Comment:
Again, I'd use `?\stdClass` instead of `void` here.

Reply:
Fixed.